src/mscorlib/shared/System/IO/FileStreamCompletionSource.Win32.cs

```diff
@@ -36,7 +36,7 @@ private unsafe class FileStreamCompletionSource : TaskCompletionSource<int>
             private long _result; // Using long since this needs to be used in Interlocked APIs
 
             // Using RunContinuationsAsynchronously for compat reasons (old API used Task.Factory.StartNew for continuations)
-            internal FileStreamCompletionSource(FileStream stream, int numBufferedBytes, byte[] bytes)
+            protected FileStreamCompletionSource(FileStream stream, int numBufferedBytes, byte[] bytes)
```

Comment:
Made this protected so that it can't be newed up. Forces callers to find the factory.

Reply:
Sounds good